.github/workflows/build.yml

```diff
@@ -267,18 +274,11 @@ jobs:
         with:
           name: extension-artifacts
 
-      - name: debug
-        run: ls
-
-      - name: Install the extension
+      - name: Install the extension artifacts
         shell: bash -l {0}
         run: |
           set -eux
-          cp ./jupytergis_core/dist/jupytergis*.whl jupytergis_core-0.0.1-py3-none-any.whl
-          cp ./jupytergis_lab/dist/jupytergis*.whl jupytergis_lab-0.0.1-py3-none-any.whl
-          cp ./jupytergis_qgis/dist/jupytergis*.whl jupytergis_qgis-0.0.1-py3-none-any.whl
-          cp ./jupytergis_lite/dist/jupytergis*.whl jupytergis_lite-0.0.1-py3-none-any.whl
-          cp ./jupytergis/dist/jupytergis*.whl jupytergis-0.0.1-py3-none-any.whl
+          pip install ./jupytergis_core/dist/jupytergis*.whl ./jupytergis_lab/dist/jupytergis*.whl ./jupytergis_qgis/dist/jupytergis*.whl ./jupytergis_lite/dist/jupytergis*.whl ./jupytergis/dist/jupytergis*.whl
```

Comment:
Not sure if it makes sense to install them all in the lite environment, but it probably doesn't hurt.

Let's see if the CI passes at least.